task_12/index.html

```diff
@@ -0,0 +1,115 @@
+                    <figcaption>
+                        <time class="pizzaItem__Timer" datetime="12:43:32">12:43:32</time>
+                        <p class="pizzaItem__Queue"><span class="pizzaItem__Queue_hash">#</span>3</p>
+                        <p class="pizzaItem__CookedTime">ETA:
+                            <time datetime="2018-02-05 18:30:45">7&nbsp;min</time>
+                        </p>
+                        <p class="pizzaItem__Price"><span class="pizzaItemPrice_moneyUnit">$</span>13</p>
+                    </figcaption>
+                </figure>
+                <figure class="pizzaItem">
+                    <img src="dist/img/pizza_im.jpg" alt="pizza img" class="pizzaItemImg">
+                    <figcaption>
+                        <time class="pizzaItem__Timer" datetime="12:43:32">12:43:32</time>
+                        <p class="pizzaItem__Queue"><span class="pizzaItem__Queue_hash">#</span>4</p>
+                        <p class="pizzaItem__CookedTime">ETA:
+                            <time datetime="2018-02-05 18:30:45">7&nbsp;min</time>
+                        </p>
+                        <p class="pizzaItem__Price"><span class="pizzaItemPrice_moneyUnit">$</span>23</p>
+                    </figcaption>
+                </figure>
+                <figure class="pizzaItem">
+                    <img src="dist/img/pizza_im.jpg" alt="pizza img" class="pizzaItemImg">
+                    <figcaption>
+                        <time class="pizzaItem__Timer" datetime="12:43:32">12:43:32</time>
+                        <p class="pizzaItem__Queue"><span class="pizzaItem__Queue_hash">#</span>5</p>
+                        <p class="pizzaItem__CookedTime">ETA:
+                            <time datetime="2018-02-05 18:30:45">5&nbsp;min</time>
+                        </p>
+                        <p class="pizzaItem__Price"><span class="pizzaItemPrice_moneyUnit">$</span>14</p>
+                    </figcaption>
+                </figure>
+                <figure class="pizzaItem">
+                    <img src="dist/img/pizza_im.jpg" alt="pizza img" class="pizzaItemImg">
+                    <figcaption>
+                        <time class="pizzaItem__Timer" datetime="12:43:32">12:43:32</time>
+                        <p class="pizzaItem__Queue"><span class="pizzaItem__Queue_hash">#</span>6</p>
+                        <p class="pizzaItem__CookedTime">ETA:
+                            <time datetime="2018-02-05 18:30:45">5&nbsp;min</time>
+                        </p>
+                        <p class="pizzaItem__Price"><span class="pizzaItemPrice_moneyUnit">$</span>23</p>
+                    </figcaption>
+                </figure>
+            </section>
+        </main>
+    </div>
+    <footer>
+        <p class="orgNameAndLocation"><span class="organization">Kottans,</span>Kottans Str. 1</p>
+        <a class="phoneNumber" href="tel:577-788-87">tel: 577 - 788 - 87</a>
```

Comment:
If you want to use spaces inside of phone number, use non-breaking ones, i.e. `&nbsp;`. If you want to use hyphens, use it without spaces.

In your case phone number may get line break (or may not) between digit parts.

Reply:
You're probably a holy man if you're ready to do a so cool code review
Anyway, great thank you

Corrected